.travis.yml

```diff
@@ -77,6 +77,13 @@ jobs:
       - export PATH="`pwd`/geckodriver:$PATH"
       script: npx nyc ./tests/training-integration.sh
       after_success: npm run coverage
+    -
+      name: "GO Tests"
+      language: go
+      go: 1.16.x
+      before_script: cd go
+      script: go test -v ./...
+      after_success: go test -cover ./...
```

Comment:
I think this is not right: as I read the manual, -cover enables coverage analysis when running the tests (like running with `nyc` in node), so this runs the tests twice.
I think we should set -cover in the normal test script, and set -coverprofile as well. In after_success, we should use [goveralls](https://docs.coveralls.io/go) to ship the cover.out file to coveralls so it gets merged with the JS code coverage and used to build reports.